modules/luci-base/luasrc/sys.lua

```diff
@@ -189,6 +189,23 @@ local function _nethints(what, callback)
 			end
 		end
 	)
+	
+	cur:foreach("dhcp", "odhcpd",
+		function(s)
+			if s.leasefile and fs.access(s.leasefile) then
```

Comment:
Check for `type(s.leasefile) == "string"` otherwise, `fs.access()` will assert for `list leasefile foo`.

Reply:
this should also be done to dnmasq function then